### PR TITLE
move list of bots into a separate file

### PR DESCRIPTION
**Pull request by DoctorLai**
it is not a good idea to maintain such a long list in a single bot file.

### CODE REVIEW

**Comment by realrufans** [comment]
Oh yea , I agree with you

File: list-of-bots.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+          'whatamidoing',
+          'whatupgg',
+          'wildoekwind',
+          'wiseguyhuh',
+          'wistoepon',
+          'zdashmash',
+          'zdemonz',
+          'zhusatriani'
+     ];
+     return bots;
```
Comment (olo2552) [inline list-of-bots.js]: One-time used variable is not necessary - simple returning array is more memory & performance efficient.

File: list-of-bots.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
+exports.bots = function () {
```
Comment (olo2552) [inline list-of-bots.js]: Why it is done this way? Isn't it more proper to use `module.exports = [];` to export a const var?

**Comment by DoctorLai** [comment]
I think we can abandon this PR since it has been more than two months and nobody reviews it. Source code has been quite a lot different.

**Pull request closed without merging**